pom.xml

```diff
@@ -126,9 +126,9 @@
         </dependency>
         
         <dependency>
-            <groupId>com.decodified</groupId>
-            <artifactId>scala-ssh_2.11</artifactId>
-            <version>0.7.0</version>
+            <groupId>com.github.seratch.com.veact</groupId>
+            <artifactId>scala-ssh_${scalaArtifactIDVersion}</artifactId>
+            <version>0.8.0-1</version>

```

Comment:
is this works fine with scala 2.11?

Reply:
@yonyyatsun that, we need to verify. But if you look at the actual source of AWScala https://github.com/seratch/AWScala/blob/master/build.sbt, they are using the same dependency and they provide support for Scala 2.10 to 2.12.

Reply:
This dependency is maintained by the same person who is maintaining AWScala. https://github.com/seratch/scala-ssh

Reply:
@harmeetsingh0013 Ok. Just test Sanity check with 2.11 (for example Report Service)

Reply:
@yonyyatsun Report Service means Dispatch Cluster?

Reply:
@harmeetsingh0013 Yes

Reply:
@yonyyatsun the problem is, on our machines ReportService is not working. Maybe you guys can help us for validation.

Reply:
Hey, @yonyyatsun I executed Dispatch Cluster on port 8081, which is running fine and hit  http://localhost:8081/SanityHealthCheck service for Sanity Health Check, which returns 200 Ok. Please see attached screenshot. If something pending or need to validate some other sanity health check, please let us know.

![image](https://user-images.githubusercontent.com/5654656/58873572-46ccc080-86e4-11e9-8483-a36b531cdb62.png)